src/components/FindMovie/FindMovie.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import './FindMovie.scss';
+
+import { MovieCard } from '../MovieCard';
+import { request } from '../../api/api';
+
+export const FindMovie = ({ onHasMovie: hasMovie, onAddMovie: addMovie }) => {
```

Comment:
you can give names to your props right when you pass them (hasMovie={hasMovie})

Reply:
good point